packages/code-du-travail-frontend/src/outils/common/NoticeExample.tsx

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+
+import { SmallText, StyledSmallText } from "./stepStyles";
+import { dateToString } from "../../lib";
+import { convertPeriodToHumanDate, Extra, getExtra } from "../utils";
+
+const FROM_DATE = new Date("2022-04-22");
+
+type NoticeExampleProps = {
+  simulator: Simulator;
+  period: string;
+  fromDate?: Date;
+  note?: JSX.Element;
+};
+
+export enum Simulator {
+  PREAVIS_DEMISSION,
+  PREAVIS_LICENCIEMENT,
+  PREAVIS_DEPART_RETRAITE,
+  PREAVIS_MISE_RETRAITE,
+}
+
+export const NoticeExample = ({
+  simulator,
+  period,
+  fromDate = FROM_DATE,
+  note,
+}: NoticeExampleProps): JSX.Element => {
+  const resultFound = React.useMemo(
```

Comment:
petite suggestion sur le naming pour le ne pas confondre avec le resultat du simulateur. 

```suggestion
  const lastDayExample = React.useMemo(
```

Reply:
je l'ai renommé en period, pour moi, c'est plus une période qu'un dernier jour